content/docs/components/serving/seldon.md

```diff
@@ -6,72 +6,18 @@ weight = 40
 
 ## Serve a model using Seldon
 
-_This section has not yet been converted to kustomize, please refer to [kubeflow/manifests/issues/10](https://github.com/kubeflow/manifests/issues/10)._
+Seldon comes installed with Kubeflow.
 
-[Seldon-core](https://github.com/SeldonIO/seldon-core) provides deployment for any machine learning runtime that can be [packaged in a Docker container](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/README.html).
+The documentation for using Seldon can be found [here](https://docs.seldon.io/projects/seldon-core/en/latest/).
 
-Install the seldon package:
+If you have a saved model on a PVC, Google Bucket or AWS S3 you can use one of the prepacked servers as discussed [here](https://docs.seldon.io/projects/seldon-core/en/latest/servers/overview.html).
```

Comment:
Some changes to improve clarity:

* Expand abbreviations when first used. So, change `PVC` to `PersistentVolumeClaim (PVC)`. (BTW should this be just `PersistentVolume (PV)`?)

* Use correct brand names to avoid confusion: change `Google Bucket` to `Google Cloud Storage bucket` (note capitalisation). 

* Similarly, change `AWS S3` to `Amazon S3`.